pyaerocom/aeroval/config/emep/emep4no_base_config.py

```diff
@@ -0,0 +1,51 @@
+from .reporting_base import get_CFG as get_EMEP_CFG
+
+
+def get_CFG(reportyear: int, year: int, model_dir: str):
+    """Basically the EMEP base config with minor changes
+    to work with EMEP4NO input. Please also refer to the
+    EMEP config documentation as that is where the bulk
+    of the configuration takes place.
+
+    Example
+    -------------
+
+    The following snippet shows how this config can be used.
+
+    >>> from pyaerocom.aeroval.config.emep.emep4no_base_config import get_CFG # doctest: +SKIP
+    >>> import pathlib
+    >>>
+    >>> if __name__ == "__main__":
+    >>>     import matplotlib.pyplot as plt
+    >>>     import pyaerocom as pya
+    >>>     from pyaerocom import const
+    >>>     from pyaerocom.aeroval import EvalSetup, ExperimentProcessor
+    >>>
+    >>>     # Customize cache dire to avoid disk quota issues.
+    >>>     # cdir = pathlib.Path("./.cache")
+    >>>     # cdir.mkdir(exist_ok=True)
+    >>>     # const.CACHEDIR = str(cdir)
+    >>>
+    >>>     cfg = get_CFG(2024, 2022, "/lustre/storeB/project/fou/kl/emep/ModelRuns/EMEP4NO/EMEP4NO_rerun_2022/")
```

Comment:
2024 is the reporting year for emep-reporting (that is why the file was named reporting-base), with yearly reports. EMEP4NO has no yearly-reports, so this option could be dropped.